app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -455,7 +483,7 @@ public void onSectionClick(View view, int position) {
         drawerLayout.closeDrawers();
       }
     });
-    return tableDrawerAdapter;
+    return mTableDrawerAdapter;
```

Comment:
Do not use Hungarian notations make sure you adhere to these guidelines:
https://github.com/kiwix/kiwix-android/blob/master/docs/codestyle.md.

Reply:
@abdulwd  Check it now. I've removed the Hungarian notation as well as resolved the merge conflicts.